sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubUnboundedSink.java

```diff
@@ -236,8 +237,8 @@ public void populateDisplayData(Builder builder) {
     private transient PubsubClient pubsubClient;
 
     private final Counter batchCounter = Metrics.counter(WriterFn.class, "batches");
-    private final Counter elementCounter = Metrics.counter(WriterFn.class, "elements");
-    private final Counter byteCounter = Metrics.counter(WriterFn.class, "bytes");
+    private final Counter elementCounter = IOMetrics.elementsWritten();
```

Comment:
Maybe? To make sure I understand, what are your thoughts around a metric such as "number of elements/bytes written to an output PCollection" (or similarly, read from an input PCollection)? Arguably, they may be implemented by the runner and use a different approach than Metrics, so maybe these are clear? I just want to make sure we won't run into conflicts there.